sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class JaegerRemoteSamplerProvider implements ConfigurableSamplerProvider {
+
+  private static final String ENDPOINT_KEY = "endpoint";
+  private static final String POLLING_INTERVAL = "pollingInterval";
+  private static final String INITIAL_SAMPLING_RATE = "initialSamplingRate";
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    String serviceName = config.getString("otel.service.name");
+    JaegerRemoteSamplerBuilder builder = JaegerRemoteSampler.builder().setServiceName(serviceName);
+    Map<String, String> params = parseArg(config.getString("otel.traces.sampler.arg"));
+
+    // Optional configuration
+    String endpoint = params.get(ENDPOINT_KEY);
+    if (endpoint != null) {
+      builder.setEndpoint(endpoint);
+    }
+    String pollingInterval = params.get(POLLING_INTERVAL);
+    if (pollingInterval != null) {
+      builder.setPollingInterval(Integer.valueOf(pollingInterval), TimeUnit.MILLISECONDS);
+    }
+    String initialSamplingRate = params.get(INITIAL_SAMPLING_RATE);
+    if (initialSamplingRate != null) {
+      builder.setInitialSampler(Sampler.traceIdRatioBased(Double.valueOf(initialSamplingRate)));
+    }
+    return builder.build();
+  }
+
+  @Override
+  public String getName() {
+    return "jaeger_remote_sampler";
+  }
+
+  // visible for testing
+  // example value endpoint=localhost:14250,pollingIntervalMs=5000,initialSamplingRate=0.25
+  static Map<String, String> parseArg(String arg) {
+    Map<String, String> paramMap = new HashMap<>();
+    String[] params = arg.split(",");
+    for (int i = 0; i < params.length; i++) {
+      String param = params[i];
+      String[] keyVal = param.split("=");
+      if (keyVal.length != 2) {
+        throw new IllegalArgumentException(
```

Comment:
can `createSampler` throw an exception? The javadoc does not prohibit it.

Reply:
I was thinking about using `ConfigurationException` but that is not public.

Reply:
exceptions thrown at this point will bubble up to the caller of auto-configuration's `initialize()` method, which I think is fine for bad configuration, since there's no way to start up an SDK in that case.